features/create_an_item.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Test post endpoint
+  As an admin
+  I want it post an item to the database with valid parameters and return a valid response
+  
+  Scenario: In order to verify that an item can be added
```

Comment:
Comment, No Change Needed: Generally, you want your scenarios to reflect the real world. So as an admin, I really don't want to "verify that an item can be added". Instead, as an admin, "I create an item and retrieve a list of items".